homework/steven/homework_3.1/testing_app.rb

```diff
@@ -0,0 +1,160 @@
+#Build a student testing app. 
+#The app should have a class of Student and the student should "login" with their email and password before they 
+#can take a test. The user should be able to take the test and receive feedback on answers. Their score should 
+#be saved and printed at the end of the test. If they score under 60 they should be prompted to take the test again. 
+
+
+########################################################
+class Student
+
+	attr_accessor :name
+	attr_accessor :password
+
+def initialize(name, password)
+
+		@name = name
+		@password = password
+end
+
+ Auth = {
```

Comment:
If you change Auth to be an array of hashes with valid users then you could simulate retrieval of users from a database